src/overview.md

```diff
@@ -183,27 +183,68 @@ To find additional APIs, see the
+## Dart: The platform {#platform}
 
 Dart's compiler technology lets you run code in different ways:
 
-* **Native compilation**: For apps targeting mobile and desktop devices,
+* **Native platform**: For apps targeting mobile and desktop devices,
   Dart includes both a Dart VM with just-in-time (JIT) compilation and
   an ahead-of-time (AOT) compiler for producing machine code.
 
-* **Web compilation**: For apps targeting the web,
+* **Web platform**: For apps targeting the web,
   Dart includes both a development time compiler (dartdevc) and
   a production time compiler (dart2js).
   Both compilers translate Dart into JavaScript.
 
 <img src="{% asset Dart-platforms.svg @path %}" width="800px" alt="An
 illustration of the targets supported by Dart">
 
-In addition, [Flutter]({{site.flutter}}) provides tooling and UI libraries to
-build UIs that run on iOS, Android, macOS, Windows, and Linux.
+The [Flutter framework]({{site.flutter}}) is a popular multi-platform UI toolkit
+that's powered by the Dart platform, and which provides tooling and UI libraries
+to build UI experiences that run on iOS, Android, macOS, Windows, Linux, and the
+Web.
 
+#### Dart Native (machine code JIT and AOT) {#native-platform}
 
-## Learning Dart
+During development, aa fast developer cycle is critical for iteration. The Dart
+VM offers a just-in-time compiler (JIT) that supports both pure interpretation
+(as required on iOS devices, for example) and runtime optimization.
+
+When apps are ready to be deployed to production — whether you're publishing to
+an app store or deploying to a production backend — the Dart AOT compiler
+enables ahead-of-time compilation to native ARM or X64 code machine code. Your
+AOT-compiled app starts instantly and runs smoothly.
+
+The AOT-compiled code runs inside an efficient Dart runtime that enforces the
+sound Dart type system and manages memory using fast object allocation and a
+[generational garbage collector.](https://medium.com/flutter-io/flutter-dont-fear-the-garbage-collector-d69b3ff1ca30)
+
+
+More information:
+* [`dart` tool for running with JIT or AOT compiling to machine code](/tools/dart-tool)
+* [Get started: command-line and server apps](/tutorials/server/get-started)
+* [Write command-line apps](/tutorials/server/cmdline)
+* [Write HTTP clients and servers](/tutorials/server/httpserver)
+
+#### Dart Web (JavaScript dev & prod) {#web-platform}
+
+Dart Web enables running Dart code on web platforms powered by
+JavaScript. With Dart Web, you compile Dart code to JavaScript code, which in
+turn runs in a browser — for example, [V8](https://v8.dev/) inside
+[Chrome](https://www.google.com/chrome/).
+
+Dart web contains both an incremental dev compiler enabling a fast developer
+cycle, and an optimizing production compiler, `dart2js`, which compiles Dart
+code to fast, compact, deployable JavaScript using techniques such as dead-code
+elimination.
+
+More information:
+* [`dartdevc` compiler](/tools/dartdevc)
```

Comment:
We seem to be missing "get started" docs for the web side. Is that intentional? (If so... should we have a bug to add them?)

Reply:
No, we have it, i just missed the link. Added now.